lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -520,15 +520,7 @@ def test_get_html_with_existing_edx_video_id(self):
         # create test profiles and their encodings
         encoded_videos = []
         for profile, extension in [("desktop_webm", "webm"), ("desktop_mp4", "mp4")]:
```

Comment:
Should these keys be defined in edx-val and imported elsewhere without using them as strings? It seems that there's a hard-coded list of different video formats kept there?